link less libraries to reduce package dependencies

The code does not use libkrb5 and libk5crypto directly, so we do not have to
link them.  This avoids a dependency on the libkrb53 package.

--- libauthen-krb5-admin-perl.orig/Makefile.PL
+++ libauthen-krb5-admin-perl/Makefile.PL
@@ -132,7 +132,7 @@
 WriteMakefile(
     DEFINE       => '',
     INC          => "-I$KRB5_INCDIR $KRB5_EXTRAINCS",
-    LIBS         => ["-L$KRB5_LIBDIR -lkrb5 -lkadm5clnt $cryptolib -lcom_err",
+    LIBS         => ["-L$KRB5_LIBDIR -lkadm5clnt -lcom_err",
                      $KRB5_EXTRALIBS],
     NAME         => 'Authen::Krb5::Admin',
     NO_META      => 1,
